yoconfigurator/base.py

```diff
@@ -11,13 +13,32 @@ def default(self, obj):
         return super(DetectMissingEncoder, self).default(obj)
 
 
-def write_config(config, app_dir):
-    path = os.path.join(app_dir, 'configuration.json')
-    with open(path, 'w') as f:
-        json.dump(config, f, indent=4, cls=DetectMissingEncoder, separators=(',', ': '))
+def _write(content, name):
+    with open(name, 'w') as f:
+        json.dump(
+            content, f, indent=2, cls=DetectMissingEncoder,
+            separators=(',', ': '))
+
+
+def write_configs(app_dir, configs):
+    """Write configurations to the applicaiton directory."""
+    for config in configs:
```

Comment:
I think given only two calls, they are prettier.

We aren't expecting to be writing many files, so this is over-engineering the code, at a cost to the API, for no real  gain.